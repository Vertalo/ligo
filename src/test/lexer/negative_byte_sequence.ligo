const a: string = -0x222;