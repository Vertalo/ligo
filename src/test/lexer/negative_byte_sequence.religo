let a = -0x222;