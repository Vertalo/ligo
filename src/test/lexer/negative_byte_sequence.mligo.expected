ligo: lexer error: Negative byte sequence.
      Hint: Remove the leading minus sign.
       {"parser_loc":"in file \"negative_byte_sequence.mligo\", line 1, characters 8-13"}
      
